ext/opcache/jit/zend_jit_internal.h

```diff
@@ -50,13 +64,17 @@
 #define ZEND_REGSET_DIFFERENCE(set1, set2) \
 	((set1) & ~(set2))
 
-#ifndef _WIN32
+#if defined (__aarch64__)
+# define ZEND_REGSET_FIRST(set)  ((zend_reg)__builtin_ctzll(set))
+# define ZEND_REGSET_SECOND(set) ((zend_reg)__builtin_ctzll(set ^ (1ull << ZEND_REGSET_FIRST(set))))
```

Comment:
I see, this is again for zend_jit_trace_begin(). I would EXCLUDE the first used register, then call ZEND_REGSET_FIRST() again. Then we won't need this SECOND macro and special section for __aarch64__.

Reply:
@dstogov  Good catch! Will update. Thanks.